compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -772,6 +784,14 @@ public void visit(BLangPackage pkgNode) {
         result = pkgNode;
     }
 
+    // Add global variables with default values to init function
+    private void addToInitFunction(BLangSimpleVariable globalVar, BLangBlockFunctionBody initFnBody) {
+        if (globalVar.expr != null) {
+            BLangAssignment assignment = createAssignmentStmt(globalVar);
+            initFnBody.stmts.add(assignment);
+        }
```

Comment:
Shall we check if the expr is null and return early instead?

```suggestion
        if (globalVar.expr == null) {
                return;
        }
        BLangAssignment assignment = createAssignmentStmt(globalVar);
        initFnBody.stmts.add(assignment);
```

Reply:
Okay